pkg/plugins/resources/file/source.go

```diff
@@ -14,7 +14,11 @@ import (
 // Source return a file content
 func (f *File) Source(workingDir string) (string, error) {
 	var validationErrors []string
+	var foundContent string
 
+	if len(f.spec.Files) > 1 {
+		validationErrors = append(validationErrors, "Validation error in source of type 'file': the attributes `spec.files` can't contain more than one element for sources")
```

Comment:
Wouldn't make sense to keep the parameter `file`? Instead of deprecating it? 
It could be transparent to the user

Reply:
I am asking because I wonder if from a UX stand point it makes sense to ask for switching from `file: changelog.md` to `files: ["changelog.md"]`

Reply:
You could also see it changing from
```
file: changelog.md
```
to
```
files:
- changelog
```
which is IMO far more than acceptable.

Also, keeping the 2 would needlessly complicate the doc and the comprehension burden for new users.

Reply:
I'm still deprecating it ;)

Reply:
Fair. However I'm not convinced that the code would be overcomplicated to keep the singular version: you are already initializing the list of files from the single argument form. It means removing the warning and the depreciation comment and that should be enough

Reply:
Of course, but why keep the 2? I don't think it's really usefull in the long term

Reply:
In this case, it's the same as `kubectl get pod` and `kubectl get pods` having the same behavior: both are doing the same thing, and avoid user having to retype commands. For me it's UX

Reply:
OK

Reply:
I've removed the deprecation warning and comments: [`751ce51` (#490)](https://github.com/updatecli/updatecli/pull/490/commits/751ce5166607701afc6329f7a8d0eada2d0aa4d8)

Reply:
For me it's more a UX thing, you are doing the right thing by converting from file to files so I would just remove the deprecation message and keep `file`

Reply:
I'll let you resolve this conversation then :)